contents/tutorials/filter-internal-users.mdx

```diff
@@ -43,6 +43,10 @@ Any internal users you identify for a project will apply for _all_ users on that
 
 ## Step 2: Add an internal user filter
 
+> **Update:** When filtering out internal users by person properties, like email, 
+> instead of the instructions below, we recommend [creating a Cohort](https://posthog.com/docs/data/cohorts#method-2-create-a-new-cohort-from-the-cohorts-page)
+> with those properties, and then adding that cohort with a "not in" operator in your ‘_Filter out internal and test users_’ settings.
```

Comment:
```suggestion
> **Update 18 Sept, 2024:** When filtering out internal users by person properties, like email, 
> instead of the instructions below, we now recommend [creating a Cohort](https://posthog.com/docs/data/cohorts#method-2-create-a-new-cohort-from-the-cohorts-page)
> with those properties, and then adding that cohort with a "not in" operator in your ‘_Filter out internal and test users_’ settings.
```